design-book/src/design-constraints.md

```diff
@@ -53,6 +54,14 @@ Below each bullet point, motivation is given.
   - Dogfooding and improving `bevy_ui` is a major goal. If something sucks, fix it upstream!
 - No scripting languages.
   - This targets a very niche audience: Rust is safer and easier to write than C++ and pure artists won't write Lua either. Prefer GUI controls, and then defer to node-based logic for highly complex art/design workflows.
+- The Bevy Editor is not a plugin for a third-party editor like Blender or LDTK. While tools like [Blenvy](https://github.com/kaosat-dev/Blenvy) can be very productive for a team making a specific game, they're not a replacement for the Bevy Editor.
+  - The existence of a standalone Bevy Editor tool (even with relatively limited capabilities) is valuable for helping new users get comfortable with the idea of making games in Bevy and become a functional member of a team.
```

Comment:
i would possibly say here that the editor will be extendable with plugins same as bevy. to off shoot limited capabilities

Reply:
Yeah, that's listed in the roadmap under uncategorized work :)